src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs

```diff
@@ -73,6 +73,20 @@ private static bool GetIsWindowsSubsystemForLinux()
             return false;
         }
 
+        static Lazy<bool> s_largeArrayIsNotSupported = new Lazy<bool>(() => {
+            try
+            {
+                var tmp = new byte[int.MaxValue];
```

Comment:
Some codegens may optimize this allocation out. I may be useful to consume the value in a way that is unlikely to be optimized out. E.g. call `ToString()` on it.